stablehlo/tools/StableHloLspServerMain.cpp

```diff
@@ -0,0 +1,22 @@
+#include "mlir/Tools/mlir-lsp-server/MlirLspServerMain.h"
```

Comment:
Nit: Let's rename to `StablehloLspServerMain.cpp` to follow the conventional spelling of StableHLO in code: "Stablehlo" (without capitalization except for the first letter as appropriate).